context.rs doesn't compile with internal-getters feature

**Describe the Bug**
Compiling with internal-getters:

```
error[E0308]: mismatched types
    --> src/context.rs:1236:9
     |
1235 |     unsafe fn get_ref(&self) -> LLVMContextRef {
     |                                 -------------- expected `*mut LLVMContext` because of return type
1236 |         self.context
     |         ^^^^^^^^^^^^ expected *-ptr, found struct `ContextImpl`
     |
     = note: expected raw pointer `*mut LLVMContext`
                     found struct `ContextImpl`

error[E0308]: mismatched types
    --> src/context.rs:2030:9
     |
2029 |     unsafe fn get_ref(&self) -> LLVMContextRef {
     |                                 -------------- expected `*mut LLVMContext` because of return type
2030 |         self.context
     |         ^^^^^^^^^^^^ expected *-ptr, found struct `ContextImpl`
     |
     = note: expected raw pointer `*mut LLVMContext`
                     found struct `ContextImpl`
```

**To Reproduce**
`cargo check -F llvm13-0,internal-getters` in this repository.

**Expected Behavior**
Inkwell should compile.

**LLVM Version (please complete the following information):**
 - LLVM Version: 13
 - Inkwell Branch Used: master

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04

**Additional Context**
Add any other context about the problem here.

--- a/src/context.rs
+++ b/src/context.rs
@@ -1233,7 +1233,7 @@ impl Drop for Context {
 #[cfg(feature = "internal-getters")]
 impl LLVMReference<LLVMContextRef> for Context {
     unsafe fn get_ref(&self) -> LLVMContextRef {
-        self.context
+        self.context.0
     }
 }
 
@@ -2027,6 +2027,6 @@ impl<'ctx> ContextRef<'ctx> {
 #[cfg(feature = "internal-getters")]
 impl LLVMReference<LLVMContextRef> for ContextRef<'_> {
     unsafe fn get_ref(&self) -> LLVMContextRef {
-        self.context
+        self.context.0
     }
 }

